__tests__/app.test.js

```diff
@@ -3,28 +3,72 @@ const app = require("../app");
 const db = require("../db/connection");
 const seed = require("../db/seeds/seed");
 const testData = require("../db/data/test-data");
+const descriptionTemplateData = require("../description-templet");
 
 beforeEach(() => {
     return seed(testData);
 });
 
 afterAll(() => {
     return db.end();
-    // return seed(testData).then(() => db.end());
 });
 
-describe("GET /api/topics", () => {
-    test("should return 200 and array of objects which they should have slug and description properties", () => {
-        return request(app)
-            .get("/api/topics")
-            .expect(200)
-            .then(({ body }) => {
-                const topics = body.topics;
-                expect(topics).toHaveLength(3);
-                topics.forEach((topic) => {
-                    expect(topic).toHaveProperty("slug", expect.any(String));
-                    expect(topic).toHaveProperty("description", expect.any(String));
+describe("GET / POST / PUT / PATCH / DELETE ", () => {
```

Comment:
This describe block should only be GET at the moment. It’s overcomplicating things right now, as we do not have a PATCH, POST, PUT or DELETE in the code yet. Please refactor this so the describe block is only representing what is currently in the code.